### PR TITLE
Improved the "No wallet" experience

### PR DESCRIPTION
**Pull request by julien51**
# Description

Per Forbes feedback, we needed a way for them to customize the "no wallet" experience.
This PR adds the ability to add a custom message.
They also asked for larger texts above the icon and smaller icons.

This needs to be rebased after #5228

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [X] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [X] My code follows the style guidelines of this project, including naming conventions
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] If my code adds or changes components, I have written corresponding stories with Storybook
- [X] I have performed a self-review of my own code
- [X] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
image

**Comment by julien51** [comment]
image

File: paywall/src/__tests__/data-iframe/Mailbox/getBlockchainDataFromLocalStorageCache.test.ts [hunk 1]
```diff
@@ -69,6 +69,7 @@ describe('Mailbox - getBlockchainDataFromLocalStorageCache', () => {
         pending: '',
         expired: '',
         confirmed: '',
+        noWallet: '',
```
Comment (julien51) [inline paywall/src/__tests__/data-iframe/Mailbox/getBlockchainDataFromLocalStorageCache.test.ts]: More paywall non-sense. I had to change all of the (unrelated) tests to make sure things worked :/

Comment (cnasc) [reply paywall/src/__tests__/data-iframe/Mailbox/getBlockchainDataFromLocalStorageCache.test.ts]: Classic

**Pull request merged**